src/update.js

```diff
@@ -0,0 +1,76 @@
+/*
+ *
+ *  CPUPower for GNOME Shell preferences
+ *  - Creates a widget to set the preferences of the cpupower extension
+ *
+ * Copyright (C) 2020
+ *     Martin Koppehel <martin.koppehel@st.ovgu.de>,
+ *     Fin Christensen <christensen.fin@gmail.com>,
+ *
+ * This file is part of the gnome-shell extension cpupower.
+ *
+ * gnome-shell extension cpupower is free software: you can
+ * redistribute it and/or modify it under the terms of the GNU
+ * General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * gnome-shell extension cpupower is distributed in the hope that it
+ * will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with gnome-shell extension cpupower.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ *
+ */
+
+// Gnome imports
+const PopupMenu = imports.ui.popupMenu;
+
+// Relative and misc imports and definitions
+const ExtensionUtils = imports.misc.extensionUtils;
+const Me = ExtensionUtils.getCurrentExtension();
+const Convenience = Me.imports.src.convenience;
+const baseindicator = Me.imports.src.baseindicator;
+const attempt_update = Me.imports.src.utils.attempt_update;
+
+const SETTINGS_ID = 'org.gnome.shell.extensions.cpupower';
+const Gettext = imports.gettext.domain('gnome-shell-extension-cpupower');
+const _ = Gettext.gettext;
+
+const updateText = _('Your CPU Power Manager installation needs updating!');
+const securityText = _('Warning: Security issues were found with your installation!\nPlease update immediately!');
```

Comment:
For some reason these two strings were not translated (to German) during my tests on Ubuntu 20.04, 19.10, 18.04 and Arch Linux with Gnome 3.36.1.

Reply:
Ah yes, I forgot the translation context. Thanks!

Reply:
Fixed in 0e41c73